src/com/mateacademy/calculator/Calculator.java

```diff
@@ -0,0 +1,5 @@
+package com.mateacademy.calculator;
+
+public interface Calculator {
+    double calculator(int firstNumber, int secondNumber, char operation);
```

Comment:
It is better to change the signature of the method. 
double calculator(int firstNumber, int secondNumber, char operation)->double calculator(double firstNumber,char operation, double secondNumber)